### DISCUSSION TITLE
answer in portuguese

### DISCUSSION THREAD

**Discussion by adrianomarqui** [logancyang/obsidian-copilot#50] [Q&A] [answered]
When I select a text in Portuguese and I choose the copilot option "Explain selection like I'm 5", it answers me in English. Can I change this?

**Comment by logancyang** [comment, marked as answer]
Thanks for pointing this out! Will implement a solution soon!

**Comment by logancyang** [comment]
@adrianomarqui Just released 2.2.1, could you update the plugin and check if it fixes your issue? https://github.com/logancyang/obsidian-copilot/issues/55

**Reply by adrianomarqui** [reply]
Wow, this is amazing, it's working. The people of the "legendary community" in Brazil are very grateful. A big hug and we will support you at buymeacoffee.com.
